types/spotify-web-playback-sdk/index.d.ts

```diff
@@ -99,6 +121,11 @@ declare namespace Spotify {
         shuffle: boolean;
         restrictions: PlaybackRestrictions;
         track_window: PlaybackTrackWindow;
+        playback_id: string;
+        playback_quality: string;
+        playback_features: {
+            hifi_status: string;
```

Comment:
do we know the possible values for this? feels like an enum to me

Reply:
I agree, although unfortunately I can't find concrete documentation listing their values (same as the disallow_*_reasons), and the minified source for the playback SDK doesn't reveal anything either.